src/main/scala/ai/datahunters/md/util/HashUtils.scala

```diff
@@ -0,0 +1,65 @@
+package ai.datahunters.md.util
+
+import java.security.MessageDigest
+import java.util.zip.{CRC32, Checksum}
+
+object HashUtils {
+
+  lazy val HashExecutionMap: Map[String, Function[Array[Byte], (String, String)]] = Map(
```

Comment:
@dawidsau  Shorter would be `(Array[Byte]) => (String, String)` instead of Function[Array[Byte], (String, String)] but  it's ok and you can leave it.